src/datasets/bigquery/exact_mau/reference.md

```diff
@@ -137,9 +137,7 @@ ORDER BY
 Additional usage criteria may be added in the future as new columns named
 `*_*mau`, etc. where the prefix describes the usage criterion.
 
-For convenience and clarity, we make the exact data presented in the
-[2019 Key Performance Indicator Dashboard](https://dbc-caf9527b-e073.cloud.databricks.com/#job/1160/run/latestSuccess/dashboard/42765b23-7b69-4d59-b08b-ea9cf45b63df)
-available as views that do not require any aggregation:
+For convenience and clarity, we make the exact data presented in the 2019 Key Performance Indicator Dashboard available as views that do not require any aggregation:
```

Comment:
I imagine this has been replaced with something else, we should get someone to update this (maybe not as part of this PR). I'll ask around.

Reply:
cc @xluo-ds @gkabbz who I believe are owners of the KPI dashboards.

Reply:
Some KPI Dashboard was migrated to Data Studio here: https://bugzilla.mozilla.org/show_bug.cgi?id=1664234.

Reply:
@akkomar is right. That was moved to the datastudio dashboard linked in the bug for 2020. And that datastudio dashboard will be moved to a looker dashboard pretty soon (once we have 2021 targets and the datasources are available in looker).